Decred Wallet/view_util/DropMenuButton.swift

```diff
@@ -23,6 +23,11 @@ class DropMenuButton: UIButton, UITableViewDelegate, UITableViewDataSource
     
     var superSuperView = UIView()
     var containerView = UIView()
+    var minTableWidth: CGFloat = 0 {
+        didSet {
+            self.layoutIfNeeded()
```

Comment:
I'm thinking `self.setNeedsLayout()` to ensure a re-layout is done. `layoutIfNeeded` will only perform a re-layout if it thinks it's necessary, changing the `minTableWidth` does not in itself make a re-layout necessary.